osi_ippay_payment/models/payment.py

```diff
@@ -0,0 +1,56 @@
+# Copyright (C) 2019 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import requests
+import xmltodict
```

Comment:
xmltodict needs to be manually installed.
We should mention this in an Installation section of the README, and ideally include it in the manifest: `'external_dependencies': {'python': ['xmltodict ']}`

Reply:
or maybe inside https://github.com/ursais/osi-addons/blob/12.0/requirements.txt also

Reply:
If you could make the requirements.txt have version numbers for the python libraries, it would allow me to add this file to my deploy automation to ensure the pip package is installed. Similar to the requirements.txt in https://github.com/odoo/odoo/blob/13.0/requirements.txt.